frontend/projects/upgrade/src/app/core/experiment-design-stepper/experiment-design-stepper.service.ts

```diff
@@ -338,7 +338,7 @@ export class ExperimentDesignStepperService {
     const payloadTableData = this.getSimpleExperimentPayloadTableData();
 
     payloadTableData.forEach((payloadRowData: SimpleExperimentPayloadTableRowData) => {
-      const parentCondition = conditions.find((condition) => condition.conditionCode === payloadRowData.condition).id;
+      const parentCondition = conditions.find((condition) => condition.conditionCode === payloadRowData.condition);
```

Comment:
this change seems unintentional

Reply:
Yeah, its the same ghost change from auto merge for which I had created 2 PR's which allowed creation of experiments. I pointed out the same in my below review as well. :)